service/src/main/java/bio/terra/workspace/app/controller/ControlledAzureResourceApiController.java

```diff
@@ -15,36 +15,7 @@
 import bio.terra.workspace.common.utils.AzureUtils;
 import bio.terra.workspace.common.utils.ControllerValidationUtils;
 import bio.terra.workspace.generated.controller.ControlledAzureResourceApi;
-import bio.terra.workspace.generated.model.ApiAzureDatabaseResource;
-import bio.terra.workspace.generated.model.ApiAzureDiskResource;
-import bio.terra.workspace.generated.model.ApiAzureKubernetesNamespaceResource;
-import bio.terra.workspace.generated.model.ApiAzureLandingZoneResourcesList;
-import bio.terra.workspace.generated.model.ApiAzureManagedIdentityResource;
-import bio.terra.workspace.generated.model.ApiAzureVmCreationParameters;
-import bio.terra.workspace.generated.model.ApiAzureVmResource;
-import bio.terra.workspace.generated.model.ApiCloneControlledAzureStorageContainerRequest;
-import bio.terra.workspace.generated.model.ApiCloneControlledAzureStorageContainerResult;
-import bio.terra.workspace.generated.model.ApiClonedControlledAzureStorageContainer;
-import bio.terra.workspace.generated.model.ApiCreateControlledAzureBatchPoolRequestBody;
-import bio.terra.workspace.generated.model.ApiCreateControlledAzureDatabaseRequestBody;
-import bio.terra.workspace.generated.model.ApiCreateControlledAzureDiskRequestBody;
-import bio.terra.workspace.generated.model.ApiCreateControlledAzureKubernetesNamespaceRequestBody;
-import bio.terra.workspace.generated.model.ApiCreateControlledAzureManagedIdentityRequestBody;
-import bio.terra.workspace.generated.model.ApiCreateControlledAzureStorageContainerRequestBody;
-import bio.terra.workspace.generated.model.ApiCreateControlledAzureVmRequestBody;
-import bio.terra.workspace.generated.model.ApiCreatedAzureStorageContainerSasToken;
-import bio.terra.workspace.generated.model.ApiCreatedControlledAzureBatchPool;
-import bio.terra.workspace.generated.model.ApiCreatedControlledAzureDatabaseResult;
-import bio.terra.workspace.generated.model.ApiCreatedControlledAzureDisk;
-import bio.terra.workspace.generated.model.ApiCreatedControlledAzureKubernetesNamespaceResult;
-import bio.terra.workspace.generated.model.ApiCreatedControlledAzureManagedIdentity;
-import bio.terra.workspace.generated.model.ApiCreatedControlledAzureStorageContainer;
-import bio.terra.workspace.generated.model.ApiCreatedControlledAzureVmResult;
-import bio.terra.workspace.generated.model.ApiDeleteControlledAzureResourceRequest;
-import bio.terra.workspace.generated.model.ApiDeleteControlledAzureResourceResult;
-import bio.terra.workspace.generated.model.ApiJobControl;
-import bio.terra.workspace.generated.model.ApiJobReport;
-import bio.terra.workspace.generated.model.ApiResourceQuota;
+import bio.terra.workspace.generated.model.*;
```

Comment:
do you need to make sure your code formatter is setup with the google format options? this change feels odd to me given the amount of models that were previously explicitly listed (which may have been the code's standard)

Reply:
This was done by the pre-configured gradle formatter not my IDE, the same gradle formatter is used as a check on PRs for consistency. 

Should be good in this regard since the same config for ensuring PR consistency is how I formatted it, not just an IDE garble.